### ISSUE TITLE
reference to non existing ModelType

### ISSUE THREAD

**Issue by cantoute** [kimjbstar/prisma-class-generator#54]
## Expected Behavior

Here using a simple model named `Node` with a parent/children tree relation.

`schema.prisma`
```prisma
// This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema

generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

generator prismaClassGenerator {
  provider = "prisma-class-generator"
  dryRun   = false
}

model Node {
  id Int @id @default(autoincrement())

  parentId Int?
  parent   Node?  @relation("NodeToNodeParent", fields: [parentId], references: [id], onDelete: Restrict)
  children Node[] @relation("NodeToNodeParent")
}
```

generated `src/_gen/prisma-class/node.ts`
```ts
import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger'

export class Node {
	@ApiProperty({ type: Number })
	id: number

	@ApiPropertyOptional({ type: Number })
	parentId?: number

	@ApiPropertyOptional({ type: () => Node })
	parent?: Node

	@ApiProperty({ isArray: true, type: () => Node })
	children: Node[]
}
```

## Actual Behavior

generated `src/_gen/prisma-class/node.ts`
```ts
import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger'

export class Node {
	@ApiProperty({ type: Number })
	id: number

	@ApiPropertyOptional({ type: Number })
	parentId?: number

	@ApiPropertyOptional({ type: () => Node })
	parent?: NodeType

	@ApiProperty({ isArray: true, type: () => Node })
	children: NodeType[]
}
```

## Steps to Reproduce the Problem

Not actually sure what triggers this problem.
On my project with a larger database model it tries to import from `./model_type` and there is no such file in the generated folder.
On the example using simply `Node` model it doesn't try to import `NodeType` but yet the problem appears as it uses it in the class.

```ts
import { Edge } from './edge';
import { FeedData } from './feed_data';
// following imports points to non existing files
import { EdgeType } from './edge_type';
import { FeedDataType } from './feeddata_type';
```

## Specifications

-   Version: 0.2.8
-   Prisma Version: 5.4.2
-   Platform: linux
-   Node: v18.17.1

**Comment by christophby** [comment]
As a quick fix you can use version 0.2.7

**Comment by AienTech** [comment]
@christophby changing to `0.2.7` didn't actually help. I can also confirm that the same happens with `0.2.6`

**Comment by kimjbstar** [comment]
fixed in 0.2.9
